### ISSUE TITLE
Exercise 6.06

### ISSUE THREAD

**Issue by elefrant1** [williamgherman/c-solutions#114]
Hi,

Worked for me to compress it even more (but can't yet say if there are other drawbacks with this approach due to being new to C):

for (int i = 1; i <= 128; printf("%d ", i), i *= 2)
____ ;

Could move the ";" to same line as well with a comment for clarity

for (int i = 1; i <= 128; printf("%d ", i), i *= 2); // Empty body

**Comment by alvinng4** [comment]
This is not wrong but this is not a good practice since it is not easy to read. I prefer
```
for (int i = 1; i <= 128; i *= 2)
{
        printf("%d ", i);
}
```
